pipeline/pipeline-dev.yml

```diff
@@ -3,55 +3,29 @@ resources:
   type: git
   source:
     uri: https://github.com/ONSdigital/census-worth-self-help.git
-    branch: master
+    branch: feature/ONS-152-pipeline-cleanup
   webhook_token: ((git.census-worth-self-help-webhook-token))
   check_every: 15m
 
+- name: content-repository
```

Comment:
test repository is not required for a dev build.  It should be built totally from the dev repository.

Reply:
reverted dev pipeline as it was doing few different steps compared to the other pipelines